docs/machine-learning/how-to-guides/machine-learning-model-predictions-ml-net.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "How-To: Use Trained Model To Make Predictions"
+description: Learn to make predictions using a trained model
+ms.date: 04/25/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to use my model to make predictions
+---
+
+# How-To: Make predictions with a trained model
+
+Learn how to use a trained model to make predictions
+
+## Data Models
+
+### Input Data
+
+```csharp
+public class HousingData
+{
+    [LoadColumn(0)]
+    public float Size { get; set; }
+
+    [LoadColumn(1, 3)]
+    [VectorType(3)]
+    public float[] HistoricalPrices { get; set; }
+
+    [LoadColumn(4)]
+    [ColumnName("Label")]
+    public float CurrentPrice { get; set; }
+}
+```
+
+### Output Data
+
+Like input columns names, ML.NET has default names for columns that contain the predicted value of a model. Depending on the task it may be different. 
```

Comment:
```suggestion
Like input column names, ML.NET has default names for columns that contain the predicted value of a model. Depending on the task it may be different. 
```